net/FlatBuffers/FlatBufferVerify.cs

```diff
@@ -107,7 +107,7 @@ public Verifier()
     {
       // Verifier buffer
       verifier_buffer = null;
-      // Verifier settings 
+      // Verifier settings
```

Comment:
I try not to touch trailing whitespace even though I don't like it, but I had to manually touch this file and VSCode insisted on removing it...